### ISSUE TITLE
NaN in outputs when adding custom loss

### ISSUE THREAD

**Issue by Eereenah** [affinelayer/pix2pix-tensorflow#27]
Hi, I have added custom loss instead of L1 but I keep getting nan values as output

`        gen_loss_custom = tf.reduce_mean(tf.sqrt(tf.abs(tf.square(targets) - tf.square(outputs))))`
Do you know what may be causing this issue? 
Thanks in advance!

**Comment by safi-manar** [comment]
It's hard to say without seeing whether you added it correctly throughout the rest of the code.
You could change `gen_loss_L1` to that, and see if that works. If so, debug from there to add it as its own loss?
